cmd/depprobcheck/Makefile

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+include ../../Makefile-modules.mk
+
+lint:
+	echo "Skipping lint for broken module depprobcheck"
```

Comment:
Seems like manual checking of a problem that can happen with the openapi dependency. It has an extensive readme, and I'm not immediately sure whether it could still be relevant, so I'd rather not remove it in this PR.

Reply:
SG, we can keep it since we aren't sure.